packages/api-server/src/__tests__/createServer.test.ts

```diff
@@ -24,9 +23,16 @@ import {
 // Set up RWJS_CWD.
 let original_RWJS_CWD: string | undefined
 
-beforeAll(() => {
+// We have to asynchronously import the module because it relies on RWJS_CWD being set.
+// TODO(jtoar): The lint rule below is complaining about the import,
+// but I'm not sure what else to do at the moment. Happy to take suggestions!
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
```

Comment:
@Tobbe merging this to get it in the RC but if you know how to fix this I'll follow up with your suggestion

Reply:
https://github.com/redwoodjs/redwood/pull/10023